### ISSUE TITLE
fix: warnings due to ambiguous `{@link}` tags when building website

### ISSUE THREAD

**Issue by radium-v** [microsoft/fast#4942]
# 🐛 Bug Report

When running `yarn generate-docs` in `sites/website`, errors for unresolvable references and ambiguous member references are emitted.

## 💻 Repro or Code Sample

```
fast/sites/website
$ yarn generate-docs

Reading fast-animation.api.json
Reading fast-colors.api.json
Reading fast-components.api.json
Reading fast-element.api.json
Reading fast-foundation.api.json

Deleting old output from docs/api
Writing @microsoft/fast-animation package
Writing @microsoft/fast-colors package
Writing @microsoft/fast-components package
WARNING: Unable to resolve reference "@microsoft/fast-foundation#AvatarTemplate": The member reference "AvatarTemplate" was not found
WARNING: Unable to resolve reference "@microsoft/fast-foundation#CardTemplate": The member reference "CardTemplate" was not found
WARNING: Unable to resolve reference "@microsoft/fast-foundation#DesignSystemProvider": The member reference "DesignSystemProvider" was not found
WARNING: Unable to resolve reference "@microsoft/fast-foundation#ToolbarTemplate": The member reference "ToolbarTemplate" was not found
WARNING: Unable to resolve reference "@microsoft/fast-foundation#AvatarTemplate": The member reference "AvatarTemplate" was not found
WARNING: Unable to resolve reference "@microsoft/fast-foundation#CardTemplate": The member reference "CardTemplate" was not found
WARNING: Unable to resolve reference "@microsoft/fast-foundation#DesignSystemProvider": The member reference "DesignSystemProvider" was not found
WARNING: Unable to resolve reference "@microsoft/fast-foundation#ToolbarTemplate": The member reference "ToolbarTemplate" was not found
Writing @microsoft/fast-element package
Writing @microsoft/fast-foundation package
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
WARNING: Unable to resolve reference "(DesignToken:interface)": The member reference "DesignToken" was ambiguous
```

## 🤔 Expected Behavior

No warnings are emitted.

## 😯 Current Behavior

Warnings are emitted.

## 💁 Possible Solution

These warnings are likely caused by `{@link}` tags in various modules that point to missing or ambiguous locations. The most common situations appear to be mismatched capitalizations for template exports, and the `DesignToken` interface not being explicit.

## 🔦 Context

These warnings may be getting caught by Docusaurus in the form of broken links. After the docusaurus build completes, this message is displayed:

```
info Docusaurus found broken links!

Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.

Exhaustive list of all broken links found:

- On source page path = /docs/components/getting-started:
   -> linking to /docs/components

- On source page path = /docs/design-systems/creating-a-design-system:
   -> linking to /docs/api/fast-foundation.designsystemprovider
   -> linking to /docs/api/fast-foundation.definedesignsystemprovider
   -> linking to /docs/api/fast-foundation.designsystemproperty
   -> linking to /docs/api/fast-foundation.designsystemprovider.custompropertymanager
   -> linking to /docs/api/fast-foundation.constructablestylescustompropertymanager

- On source page path = /docs/design-systems/fast-frame:
   -> linking to /docs/api/fast-components.fastdesignsystem
   -> linking to api/fast-components.fastdesignsystemprovider.accentpalette.md (resolved as: /docs/design-systems/api/fast-components.fastdesignsystemprovider.accentpalette.md)
   -> linking to api/fast-components.fastdesignsystemprovider.neutralpalette.md (resolved as: /docs/design-systems/api/fast-components.fastdesignsystemprovider.neutralpalette.md)
   -> linking to api/fast-components.fastdesignsystemprovider.backgroundcolor.md (resolved as: /docs/design-systems/api/fast-components.fastdesignsystemprovider.backgroundcolor.md)

- On source page path = /docs/design-systems/overview:
   -> linking to /docs/api/fast-foundation.designsystemprovider

- On source page path = /docs/design-systems/using-the-design-system:
   -> linking to /docs/api/fast-foundation.designsystemprovider
   -> linking to /docs/api/fast-foundation.csscustompropertybehavior

- On source page path = /docs/design/localization:
   -> linking to /docs/api/fast-foundation.directionalstylesheetbehavior/
   -> linking to /docs/api/fast-components.fastdesignsystemprovider.direction/
```

## 🌍 Your Environment

* OS & Device: Windows 10 and Ubuntu 20.04 (WSL2),
* Node v14.17.3

**Comment by chrisdholt** [comment]
@nicholasrice I believe this was fixed with #4970 - is that accurate or is this still outstanding?

**Comment by chrisdholt** [comment]
closing as fixed with #4970
